### PR TITLE
Deploy 67 - Updated OUSD vault value checker

### PR DESCRIPTION
**Pull request by sparrowDom**
Deploy updated OUSD vault value checker to address: [0x25041Ccb8352b9BA65883a94685aeF07a5aD8739](https://etherscan.io/address/0x25041Ccb8352b9BA65883a94685aeF07a5aD8739#code).

At least 2 full verifiers need to check that the deployed code matches the one from the repository.

### CODE REVIEW

**Review by naddison36 — APPROVED** [review]

**Review by DanielVF — APPROVED** [review]
- [x] All deployed contracts are listed in the deploy PR's description
- [x] Deployed contract's verified code (and all dependencies) match the code in master
- [x] The transactions that interacted with the newly deployed contract match the deploy script.
- [x] (N/A) _Governance proposal matches the deploy script_
- [ ] Smoke tests pass after fork test execution of the governance proposal

_Smoke testing now..._

File: brownie/allocations.py [hunk 1]
```diff
@@ -285,10 +285,10 @@ def auto_check_snapshot():
 
     return [
         world.vault_value_checker.checkDelta(
-            vault_change - 500 * int(1e18),
-            vault_change + 1000 * int(1e18),
-            supply_change - 1000 * int(1e18),
-            supply_change + 500 * int(1e18),
+            vault_change, # expectedProfit
```
Comment (DanielVF) [inline brownie/allocations.py:288]: I don't think this is how we calculate profit

File: brownie/allocations.py [hunk 2]
```diff
@@ -285,10 +285,10 @@ def auto_check_snapshot():
...
+            500 * int(1e18), # profitVariance
+            supply_change, # expectedVaultChange
```
Comment (DanielVF) [inline brownie/allocations.py:290]: Definitely not how we calc vault change

File: brownie/allocations.py [hunk 3]
```diff
@@ -285,10 +285,10 @@ def auto_check_snapshot():
...
+            500 * int(1e18), # vaultChangeVariance
```
Comment (DanielVF) [inline brownie/allocations.py:291]: While we want to keep profit / loss tight, the vault can change quite a bit and us still be okay.

**Comment by sparrowDom** [comment]
Thanks for flagging this @DanielVF. Here is the PR addressing the issue: https://github.com/OriginProtocol/origin-dollar/pull/1647/files

File: brownie/allocations.py (hunk 3 above)
Comment (sparrowDom) [reply brownie/allocations.py:291]: have set it to 1000 dollar units.

**Pull request merged**